pkg/scheduler/framework/plugins/examples/multipoint/multipoint.go

```diff
@@ -37,14 +37,25 @@ func (mc CommunicatingPlugin) Name() string {
 	return Name
 }
 
+type contextData struct {
+	Data string
```

Comment:
Does it need to be public?

Reply:
Yes, we need to check if the value in line 72

Reply:
ok, I updated it to be private, thought that this is private only for the struct methods (like C++) but it is actually private at the package level.